student-store-express-api/routes/store.js

```diff
@@ -11,7 +11,7 @@ router.get('/', (req, res) => {
 });
 
 router.get('/:productId', (req, res) => {
-	const productId = req.params.productId;
+	const productId = parseInt(req.params.productId);
```

Comment:
Could this be enforced at the express route level? I don't know for sure but I wouldn't be surprise if express has a way to enforce this at the route definition level, i.e. productId has to be an integer/number.